File: app/src/main/java/com/kacper/and_krakgo/screens/sign_in/reset_password/ResetPasswordContract.java
package com.kacper.and_krakgo.screens.sign_in.reset_password;

/**
 * Created by kacper on 05/11/2017.
 */

public interface ResetPasswordContract {
    interface View{
        void showError(String message);
        void onResetSuccesful();
    }
    interface Presenter{
        void sendResetPassword(String email);
    }
}

File: app/src/main/java/com/kacper/krakgo/screens/sign_in/reset_password/ResetPasswordContract.kt
package com.kacper.krakgo.screens.sign_in.reset_password

import com.kacper.krakgo.mvp.MvpPresenter
import com.kacper.krakgo.mvp.MvpView

object ResetPasswordContract {

    interface View : MvpView {
        fun onResetSuccessful()
    }

    interface Presenter : MvpPresenter<View> {
        fun sendResetPassword(email: String)
    }
}